Packs/MicrosoftTeams/ReleaseNotes/1_5_14.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Microsoft Teams
+
+- Updated the Microsoft Teams integration to resolve direct messages to Bot using email, upn and username.
```

Comment:
Updated the Microsoft Teams integration to resolve direct messages to the Microsoft Teams Bot using email, UPN, and username.